_docs-sources/refarch/configuration/preflight-checks.md

```diff
@@ -0,0 +1,36 @@
+# Iterate on Preflight checks
+
+Once you have run the bootstrap script and pushed your `ref-arch-form` branch with your changes, your `infrastructure-live` repository GitHub Actions will commence, running
+preflight checks against every commit that you push up to this branch.
+
+![Gruntwork Reference Architecture preflight checks](/img/preflight1.png)
+
+Preflight checks can take up to 4-5 minutes to complete after you push your commit. Any errors will be
+directly annotated on the exact line of your form that presents a blocking issue, so be sure to check the *Files changed* tab of your pull request to see them:
+
+![Gruntwork Ref Arch preflight checks on your pull request](/img/preflight-error-on-pr.png)
+
+## Fix up any errors
+
+In most cases, the error messages included in the preflight check annotations will provide sufficient information to remediate the underlying issue. If at any point you are confused or
+need assistance, please reach out to us at `support@gruntwork.io` and we'll be happy to assist you.
+
+## Commit your form changes and push to your `ref-arch-form` branch
+
+Once you have fixed any issues flagged by preflight checks, you can make a new commit with your latest form changes and push it up to the same branch. This will trigger a re-run of preflight
+checks using your latest form data.
+
+## Merge your pull request
+
+In most cases, you will require a couple of iterations in order to clear all preflight check errors. Once your preflight checks pass, meaning there are no more error annotations on your pull request
```

Comment:
```suggestion
Once your preflight checks pass, meaning there are no more error annotations on your pull request
```
We're hoping this won't be most cases especially if we're successful in funneling people through the cli